downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -0,0 +1,42 @@
+[id="aap-controller-backup"]
+
+= Backing up the {ControllerNameStart} deployment
+
+[role=_abstract]
+Use this procedure to back up a deployment of the controller, including jobs, inventories, and credentials.
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been installed to the cluster.
+* The {ControllerName} is deployed to using the {OperatorPlatform}.
+
+.Procedure
+. Log in to *{OCP}*.
+. Navigate to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Select the *Automation Controller Backup* tab.
```

Comment:
Should this one follow the accessibility example:
```suggestion
. Select the btn:[Automation Controller Backup] tab.
```

Reply:
> Should this one follow the accessibility example or does it only apply to buttons?

No, tabs and other UI elements can use the * bold *. Unfortunately, AsciiDoctor only provides macros for buttons and menus but they should only be used for button and menu items.